src/bertron_schema/schema/bertron_schema.yaml

```diff
@@ -1,96 +1,288 @@
+        from_schema: http://w3id.org/mixs/terms
+        aliases:
+          - elevation
+        slot_uri: MIXS:0000093
+        range: QuantityValue
+        multivalued: false
+      latitude:
+        range: decimal degree
+        description: latitude
+        slot_uri: wgs84:lat
+        examples:
+          - value: "-33.460524"
+        mappings:
+          - schema:latitude
+        required: true
+      longitude:
+        range: decimal degree
+        description: longitude
+        slot_uri: wgs84:long
+        examples:
+          - value: "150.168149"
+        mappings:
+          - schema:longitude
+        required: true
+    description: The coordinates defining the position associated with the entity.
 
-  DatasetCollection:
-    tree_root: true
-    description: >-
-      A holder for Dataset objects
+  Name:
+    description: The name or label for an entity. This may be a primary name,
+      alternative name, synonym, acronym, or any other label used to refer to an
+      entity.
     attributes:
-      entries:
-        range: Dataset
-        multivalued: true
-        inlined: true
+      name_type:
+        description: Brief description of the name and/or its relationship to the entity.
+        range: NameType
+        multivalued: false
+      name:
+        description: The string used as a name.
+        required: true
+        slot_uri: schema:name
+        examples:
+        - value: Heat-inducible transcription repressor HrcA
+        - value: FW106 groundwater metagenome
 
-slots:
-  id:
-    identifier: true
-    slot_uri: schema:identifier
-    range: uriorcurie
-    description: A unique identifier for a thing
-  name:
-    slot_uri: schema:name
-    description: A human-readable name for a thing
-  description:
-    slot_uri: schema:description
-    description: A human-readable description for a thing
-  primary_email:
-    slot_uri: schema:email
-    description: The main email address of a person
-  birth_date:
-    slot_uri: schema:birthDate
-    range: date
-    description: Date on which a person is born
-  age_in_years:
-    range: integer
-    description: Number of years since birth
-  vital_status:
-    range: PersonStatus
-    description: living or dead status
+  DataCollection:
+    description: Administrative unit (e.g. project, proposal, etc.) in which one or more entities is collected.
+    comments:
+      - May be equivalent to FundingReference in the CreditMetadata schema.
+    attributes:
+      id:
+        aliases:
+        - proposal ID
+        - project ID
+        description: The unique ID used for the project within the BER resource. It may not necessarily be resolvable outside the resource.
+        comments:
+        - If the data source does not use CURIEs, we cannot guarantee that IDs will be unique between all the BER sources.
+        # identifier: true
+        # required: true
+        range: string
+        slot_uri: schema:identifier
+      title:
```

Comment:
For citing samples that are part of a project/study/etc., the project title (this field) would presumably be used instead of the sample name.